torch/_dynamo/tensor_version_op.py

```diff
@@ -13,13 +13,13 @@
 
 
 @_tensor_version.py_impl(FakeTensorMode)
-def _tensor_version_fake(self):
+def _tensor_version_fake(fake_mode, self_tensor):
```

Comment:
This op goes through `_get_dispatch` . We unify the handler signature of dispatch modes to handler(mode, *args, **kwargs) to be consistent with  higher order op handler and torch bind handler.

Reply:
sgtm